themes/default/content/blog/pulumi-release-notes-m58/index.md

```diff
@@ -0,0 +1,320 @@
+          stack-name: dev
+        env:
+          PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+Learn more in the [GitHub Actions User Guide]({{< relref "/docs/guides/continuous-delivery/github-actions" >}})
+
+### Kubernetes: new `skipAwait` option
+
+For some Kubernetes use cases, including Helm charts, you may want to be able to turn off all await logic, so that using Kubernetes via Pulumi feels closer to the default Kubernetes API experience. This isn't right in all cases and has some downsides (outputs are not populated and incorrect deployments aren't identified), but for many use cases is the initially "expected" behavior. Now, with the `skipAwait` setting, you can opt out of await logic for an entire Pulumi Kubernetes program.
+
+Learn more at [pulumi/pulumi-kubernetes#1421](https://github.com/pulumi/pulumi-kubernetes/issues/1421)
+
+### 10 new resources in the Azure Native provider
+
+We shipped 5 new versions of the Azure Native provider (1.11.0 through 1.15.0) that collectively added 10 new resources that you can manage with the Azure Native provider.
+
+[See the full list](https://github.com/pulumi/pulumi-azure-native/blob/master/CHANGELOG.md#1150-2020-07-01)
+
+### Google Native provider adds support for enums and more
+
+The preview of the Google Native provider continues to evolve. We shipped 2 new versions of the Google Native provider (0.3.0 and 0.4.0) that include:
+
+- Refined, shorter resource names that are easier to type and easier to find
+- Enum support for all properties exposed as enums in the underlying Google Discovery Documents
+
+[See the full list of updates](https://github.com/pulumi/pulumi-google-native/blob/master/CHANGELOG.md#040-2021-06-16)
+
+## Pulumi CLI and core technologies
+
+In this milestone, we shipped Pulumi versions [3.5.1](https://github.com/pulumi/pulumi/releases/tag/v3.5.1) and [3.6.0](https://github.com/pulumi/pulumi/releases/tag/v3.6.0). The full list of changes in each version is available in the linked changelog; read on to learn about some of the biggest changes.
+
+### `pulumi watch` now supports a `--path` option
+
+Previously, the `pulumi watch` command always watched the entire Pulumi project directory (recursively). This could cause duplicate updates for developers working in monorepos or developers with application code and infrastructure code in the same directory. Now, you can add a `--path` option to the `pulumi watch` command and specify the exact paths that need to be watched.
+
+```shell
+pulumi watch --path path/to/pulumi/code/
+```
+
+Learn more at [pulumi/pulumi#6620](https://github.com/pulumi/pulumi/issues/6620)
+
+### Automation API: support for custom loggers in .NET
+
+Many mature projects already have a preferred tool for logging events and exceptions, but previously, Automation API only supported a built-in logger. Now, if you're using a .NET language like C#, you can supply your own logger, like Serilog.
```

Comment:
the first sentence reads a little strangely, though I get what it's intending to say

Reply:
```suggestion
Many mature projects already have a preferred tool for logging events and exceptions. Previously, Automation API only supported a built-in logging tool, and it wasn't possible to use a project's preferred tool. Now, if you're using a .NET language like C#, you can supply your own logging tool, like Serilog.
```